README.md

```diff
@@ -1,72 +1,80 @@
+technology. It transparently supports all the mechanisms associated with parallelization and distribution of workload
+across the nodes (e.g., mapping of simulation objects on different kernel instances) and optimistic synchronization (
+e.g., state recoverability). Distributed simulations rely on MPI3. In particular, global synchronization across the
+different nodes relies on asynchronous MPI primitives, for increased efficiency.
 
-ROOT-Sim's development started as a research project late back in 1987, and is currently maintained by the High Performance and Dependable Computing Systems group, a joint research group between Sapienza, University of Rome and University of Rome "Tor Vergata".
+The programming model supported by ROOT-Sim allows the simulation model developer to use a simple application-callback
+function named `ProcessEvent()` as the event handler, whose parameters determine which simulation object is currently
+taking control for processing its next event, and where the state of this object is located in memory. An object is a
+data structure, whose state can be scattered on dynamically allocated memory chunks, hence the memory address passed to
+the callback locates a top level data structure implementing the object state-layout.
 
-## Dependencies
+ROOT-Sim's development started as a research project late back in 1987, and is currently maintained by the High
+Performance and Dependable Computing Systems group, research group of the University of Rome "Tor Vergata".
 
-To build the project you need a C11 compiler such as GCC 8 or a later version, and the Meson build system.
+## ROOT-Sim Core
 
-## Building and installing
+This repository keeps the sources of the ROOT-Sim core: this is the fundamental library that implements the largest
+part of the simulation algorithms used in the simulation framework.
 
-Run:
+The core can be built and used as a stand-alone low-level library writing C code, or it can be used within other
+projects, such as [cROOT-Sim](https://gihub.com/ROOT-Sim/cROOT-Sim), i.e. the C/C++ version of the simulation library.
 
-```bash
-meson build -Dprefix={installdir}
-cd build
-ninja test
-ninja install
-```
+## Dependencies and platforms
 
-where `{installdir}` is your preferred installation directory expressed as an absolute path.\
-Alternatively you can skip the `-Dprefix` altogether: ROOT-Sim will be installed in your system directories.
+The core successfully compiles on x86 and ARM architectures, using either GCC or Clang compilers, on Linux, Windows,
+and macOS.
+A compiler supporting the C11 standard is required, such as GCC 8 or later. MSVC on Windows does not properly implement
+the full C11 standard (e.g., `stdatomic.h` is not provided), and cannot be therefore used to build the project.
 
-## Compile and run a model
+MPI is a madatory dependency of the project, used to support simulations run on distributed systems.
```

Comment:
Ooops, typo still present (and still not important)

Reply:
I don't see the typo in the file in the branch 🤔

Reply:
Yep, I just checked, you are right. Very weird indeed.